src/components/manage/Tiles/Text/Edit.jsx

```diff
@@ -19,6 +19,7 @@ import { Icon } from '../../../../components';
 import trashSVG from '../../../../icons/delete.svg';
 import addSVG from '../../../../icons/circle-plus.svg';
 import cameraSVG from '../../../../icons/camera.svg';
+import semaphoreSVG from '../../../../icons/semaphore.svg'
```

Comment:
I know that in the pastanaga icons we don't have any other better option, but instead of the semaphore I would go for the globe.svg better. I would add a list of icons that would be great to have from Albert.